cves/kernel/CVE-2021-3640.yml

```diff
@@ -153,16 +154,17 @@ discovered:
     originally found. Answer in longform below in "answer", fill in the date in
     YYYY-MM-DD, and then determine if the vulnerability was found by a Google
     employee (you can tell from their email address). If it's clear that the
-    vulenrability was discovered by a contest, fill in the name there.
+    vulnerability was discovered by a contest, fill in the name there.
 
     The automated, contest, and developer flags can be true, false, or nil.
 
     If there is no evidence as to how this vulnerability was found, then please
     explain where you looked.
-  answer:
-  automated:
-  contest:
-  developer:
+  answer: |
+    Looked in Red Hat Bugzilla report for who reported the bug. 
```

Comment:
I'm unsure what you mean by this sentence. What does the bug report say about the vulnerability?